### ISSUE TITLE
"Parsing error: Unexpected token *" with v8.2.2 in node 6

### ISSUE THREAD

**Issue by kedromelon** [babel/babel-eslint#605]
not really sure what's going on here, but i was at least able to create a isolated reproduce of it and wanted to make sure it was reported in some capacity!

>node@6.14.1
>npm@3.10.10

(doesn't seem exclusive to 6.14.1 in particular, was just what i was on when i found the issue -- i did **not** encounter the problem when trying it in node 7 or 8)

```sh
git clone git@github.com:kedromelon/babel-eslint-bug.git
npm i
npm run lint
```

```
babel-eslint-bug/index.js
  0:0  error  Parsing error: Unexpected token *

✖ 1 problem (1 error, 0 warnings)
```

specifically appears to be introduced in 8.2.2. downgrading to 8.2.1 resolves the issue.

**Comment by bary12** [comment]
see babel/babel#7654 . Should be fixed now. On a related note, since babel 7 is on beta, it should be saved as an exact version, which it currently isn't in this project.

**Comment by kedromelon** [comment]
ah great! thanks for following up
